client/src/main/scala/drt/client/components/TippyJSComponent.scala

```diff
@@ -39,7 +40,7 @@ object TippyJSComponent {
     var onTrigger: js.Function2[TippyElement, Event, Unit] = js.native
   }
 
-  def props(content: js.Object, interactive: Boolean, plugins: js.Array[js.Any], triggerEvent: String): Props = {
+  def props(label: String, content: js.Object, interactive: Boolean, plugins: js.Array[js.Any], triggerEvent: String): Props = {
```

Comment:
`label` is a little misleading as you would expect this to be shown on the page. If it's a description for ga then let's call it something mentioning analytics, eg `gaEventLabel`